proto/backend.proto

```diff
@@ -124,6 +124,8 @@ message CollectMetrics {
 
 message CheckHealth {
   message Request {
+    PluginConfig config = 1;
+    User user = 2;
```

Comment:
I'm not sure this is needed but it could be useful if plugins require certain roles or kinds of users.

Reply:
what would the user be?  I assume it should be the system user?

Reply:
Some datasources do permissions checks per user. Which would require this but I think we can skip it for now. We can always add it later.